pinc/wordcheck_engine.inc

```diff
@@ -251,7 +251,7 @@ function get_bad_words_for_text($text, $languages, $word_lists = [])
 
 class BadWordAccumulator
 {
-    /** @var string[] */
+    /** @var array<string, int> */
```

Comment:
This was mistyped. To convince oneself, just look at how `add_bad_words` inserts words:

```
$this->words[$word] = $val;
```